vendor/github.com/ethereum/go-ethereum/swarm/storage/chunker.go

```diff
@@ -50,7 +50,6 @@ data_{i} := size(subtree_{i}) || key_{j} || key_{j+1} .... || key_{j+n-1}
  The underlying hash function is configurable
 */
 
-
```

Comment:
We have a separate procedure of modifying go-ethereum: https://www.notion.so/status/Modifying-go-ethereum-245a078dde834b87aa7e4e9b4e8cc6ec
But can we avoid editing these files?

Reply:
Mistake - fmt on this file. I'll revert that.

Reply:
Develop has been merged.